psydac/feec/multipatch/examples/h1_source_pbms_conga_2d.py

```diff
@@ -263,29 +227,31 @@ def lift_u_bc(u_bc):
         PM.close()
 
     if u_ex:
-        err = uh_c - u_ex_c
-        rel_err = np.sqrt(np.dot(err, H0_m.dot(err)))/np.sqrt(np.dot(u_ex_c,H0_m.dot(u_ex_c)))
-        
+        err = u - u_ex
+        rel_err = np.sqrt( err.inner(H0.dot(err))) / np.sqrt(u_ex.inner(H0.dot(u_ex)))
```

Comment:
Here you can use the function `dot_inner` of every `LinearOperator` (see [here](https://github.com/pyccel/psydac/blob/776ac29310d1d1e1d375f7035ad5f9956ee39729/psydac/linalg/basic.py#L468-L502)) which avoids temporary vectors:
```suggestion
        rel_err = np.sqrt(H0.dot_inner(err) / H0.dot_inner(u_ex))
```